Yura_M/HW8/Task_2.py

```diff
@@ -0,0 +1,20 @@
+import re
```

Comment:
good job
look at this implementation

import re
from getpass import getpass

while True:
    password = getpass("Enter your password: ")
    if not password.strip():
        print("Error: Password cannot be blank.")
        continue
    if len(password) < 6 or len(password) > 16:
        print("Error: Password length must be between 6 and 16 characters.")
        continue
    if not re.search(r"[a-z]", password) or not re.search(r"[A-Z]", password) or not re.search(r"\d", password):
        print("Error: Password must contain at least one lowercase letter, one uppercase letter, and one digit.")
        continue
    if not re.search(r"[$#@]", password):
        print("Error: Password must contain at least one of the following characters: $ # @")
        continue
    print("Congratulations, your password is valid!")
    break